src/gwt/src/org/rstudio/studio/client/common/spelling/TypoJsEscaped.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * TypoJsEscaped.js
+ *
+ * Copyright (C) 2009-19 by RStudio, Inc.
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+package org.rstudio.studio.client.common.spelling;
+
+// This is the compressed version of the typo.js that shares the package with this file.
+// If Typo.js is ever altered or updated this file will also need to be updated. Any reasonable
+// Minifier/Compressor is fine, this is only for the PrefetchWorker to run.
+public class TypoJsEscaped
```

Comment:
If you need access to raw JS like this, you should usually use a ClientBundle, which will let you leave the `.min.js` file raw on disk (probably what you want).

Reply:
I was thinking about doing the same thing as I did with the dictionary thing which I suppose is fine since it's all local, just feels a little weird/extra work-y but I guess it is what it is. I'll look into the ClientBundle solution and do whichever feels cleaner. I agree that this is subideal.